lib/osf-components/addon/components/institutions-widget/styles.scss

```diff
@@ -9,3 +9,9 @@
         font-size: 17px;
     }
 }
+
+.SelectList {
+    button:focus {
+        outline: 0;
```

Comment:
Buttons should have outline (or some visual cue) on focus, it's important for accessibility. Why remove it?